### PR TITLE
Table header corners border-radius added

### PR DESCRIPTION
**Pull request by Sandeshyc**
Table header corners border-radius added

### CODE REVIEW

File: scss/product/components/_tables.scss [hunk 1]
```diff
@@ -10,13 +10,19 @@
   border-radius: $border-radius;
   background-color: $table-bg; // Reset for nesting within parents with `background-color`.
   thead{
-    background: $table-thead-bg-color;
+    th:first-child{
```
Comment (is-ashish) [inline scss/product/components/_tables.scss:13]: @Sandeshyc  Can you verify once in design. Seems radius have `border-top-left-radius:4px;` and `border-top-right-radius:4px;`. 

cc: @Pallavi-PH

Comment (Sandeshyc) [reply scss/product/components/_tables.scss:13]: no @is-ashish its 8px
Screenshot 2020-10-27 at 10 41 07 AM

Comment (is-ashish) [reply scss/product/components/_tables.scss:13]: I was checking in generic new table style in figma. There is `border-radius: 4px 4px 0px 0px;` cc: @Pallavi-PH 
Screenshot from 2020-10-27 12-06-00

Comment (Pallavi-PH) [reply scss/product/components/_tables.scss:13]: In style guide new table has border radius of `8px 8px 0px 0px` @is-ashish 
Screenshot 2020-10-27 at 2 38 34 PM

**Review by Pallavi-PH — APPROVED** [review]

File: scss/product/components/_tables.scss (hunk 1 above)
Comment (is-ashish) [reply scss/product/components/_tables.scss:13]: Okay, Border radius of `8px 8px 0px 0px` is fine here. But in first table have `4px 4px 0px 0px`.This is generic de-code table changes so I wanted to confirm here.

**Review by is-ashish — APPROVED** [review]

File: scss/product/components/_tables.scss (hunk 1 above)
Comment (Pallavi-PH) [reply scss/product/components/_tables.scss:13]: The first table border radius is diffrent from the other 3 mentioned in style guide. We can ask Victoria which one implement @is-ashish @Sandeshyc

**Review by Pallavi-PH — CHANGES_REQUESTED** [review]
nested child table header color is different @Sandeshyc @is-ashish 

It needs to be this color 
Screenshot 2020-10-27 at 2 59 30 PM

**Comment by is-ashish** [comment]
Re:nested child table header color is different
Will raise a separate my own PR. @Pallavi-PH . we can merged @Sandeshyc PR.

**Comment by Pallavi-PH** [comment]
> Re:nested child table header color is different
> Will raise a separate my own PR. @Pallavi-PH . we can merged @Sandeshyc PR.

Ok.

**Review by Pallavi-PH — APPROVED** [review]

**Pull request merged**